composite-view/README.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: pattern
+title: Composite View 
+folder: composite-view
+permalink: /patterns/composite-view/
+categories: structural
+language: en
+tags:
+- Enterprise Integration Pattern
```

Comment:
We should add `Presentation` tag

Reply:
will do, I will implement these suggestions.